src/ast/value.rs

```diff
@@ -73,14 +94,25 @@ impl fmt::Display for Value {
             Value::Number(v, l) => write!(f, "{}{long}", v, long = if *l { "L" } else { "" }),
             Value::DoubleQuotedString(v) => write!(f, "\"{}\"", escape_double_quote_string(v)),
             Value::SingleQuotedString(v) => write!(f, "'{}'", escape_single_quote_string(v)),
+            Value::TripleSingleQuotedString(v) => {
+                write!(f, "'''{}'''", escape_single_quote_string(v))
```

Comment:
Do we need an `escape_triple_single_quoted_string()`? Per their docs it looks like a single `'` is allowed in one of these unescaped.

Reply:
Yeah good catch! I think that some special handling is needed here when displaying the string, will take a closer look

Reply:
Updated, the unescaped quotes are now left as is